spring-boot-project/spring-boot-docs/src/docs/asciidoc/web/spring-security.adoc

```diff
@@ -223,7 +223,19 @@ Again, the same properties are applicable for both servlet and reactive applicat
 
 Alternatively, you can define your own `OpaqueTokenIntrospector` bean for servlet applications or a `ReactiveOpaqueTokenIntrospector` for reactive applications.
 
+The OAuth2 resource server now supports validating audience (aud) claims in JSON Web Tokens (JWTs) issued by an authorization server. 
+This feature is added to complement the existing validation of issuer (iss) claims.
 
+To enable audience validation, set the `spring.security.oauth2.resourceserver.jwt.audience` property in your Spring Boot application 
+configuration file. This property specifies the expected value(s) of the aud claim in JWTs.
+
+For example, to expect the JWTs to contain an aud claim with the value `my-audience`, you can add the following line to your 
+application.properties file:
+
+[source,yaml,indent=0,subs="verbatim",configprops,configblocks]
+----
+	spring.security.oauth2.resourceserver.jwt.audience=my-audience
```

Comment:
This isn't valid YAML and the property name is incorrect. It should be:

```
	spring:
	  security:
	    oauth2:
	      resourceserver:
	        jwt:
	          audiences:
	            - "my-audience"
```

The build fails due to the invalid YAML. Please build your changes (`./gradlew spring-boot-project:spring-boot-docs:asciidoctor` is sufficient in this case) before submitting them.

Reply:
Thank you so much. I have made the changes that you suggested. 

> The build fails due to the invalid YAML. Please build your changes (`./gradlew spring-boot-project:spring-boot-docs:asciidoctor` is sufficient in this case) before submitting them.

Is it possible to build within the remote repository? I made the last few commits in the remote and having a hard time syncing it with my local.

Reply:
Sorry, I don't understand what you mean by "build within the remote repository".

You can get your latest changes onto your local machine as follows:

```
$ git clone https://github.com/bikash30851/spring-boot
$ cd spring-boot
$ git checkout doc-aud-auth-issue#34848
```

Reply:
I apologize for not being clear. I was wondering if it was possible to build using GitHub actions/workflows within the remote. Because I had previously cloned it on my Windows machine but I keep getting the "Filename too long" error.